### PR TITLE
Document Image Tags in the README

### PR DESCRIPTION
**Pull request by rthallisey**

### CODE REVIEW

**Comment by rthallisey** [comment]
[Create automated builds of 'ansible-service-broker' and fusor/apb-examples.  Builds will be from 'source', (non-rpm), will use the 'canary' tag.](https://trello.com/c/CwyL8WnU/375-create-automated-builds-of-ansible-service-broker-and-fusor-apb-examples-builds-will-be-from-source-non-rpm-will-use-the-canary)

**Comment by rthallisey** [comment]
[Create automated builds of 'ansible-service-broker' and fusor/apb-examples.  Builds will be from 'source', (non-rpm), will use the 'canary' tag.](https://trello.com/c/CwyL8WnU/375-create-automated-builds-of-ansible-service-broker-and-fusor-apb-examples-builds-will-be-from-source-non-rpm-will-use-the-canary)

File: README.md [hunk 1]
```diff
@@ -70,6 +70,13 @@ as desired.
 See the [Broker Configuration](docs/config.md) doc for other example
 configurations.
 
+## Published Images
+The ansible-service-broker community publishes images in (dockerhub)[https://hub.docker.com/u/ansibleplaybookbundle].
+Image tags:
+- **canary** - The newest source build
+- **latest** - The newest RPM build
```
Comment (jmrodri) [inline README.md]: latest will specifically point to one of the release builds.

**Comment by jmrodri** [comment]
ACK

**Pull request merged**